com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs

```diff
@@ -2764,30 +1896,21 @@ void PrepareGPUProbeData(CommandBuffer cmd, HDCamera hdCamera, CullingResults cu
                         // Local Volumetric Fog are not lights and therefore should not affect light classification.
                         LightFeatureFlags featureFlags = 0;
                         CreateBoxVolumeDataAndBound(localVolumetricFogList.bounds[i], LightCategory.LocalVolumetricFog, featureFlags, worldToViewCR, 0.0f, out LightVolumeData volumeData, out SFiniteLightBound bound);
-                        m_lightList.lightsPerView[viewIndex].lightVolumes.Add(volumeData);
-                        m_lightList.lightsPerView[viewIndex].bounds.Add(bound);
+
+                        m_GpuLightList.AddLightBounds(viewIndex, bound, volumeData);
                     }
                 }
 
-                m_TotalLightCount = m_lightList.lights.Count + m_lightList.envLights.Count + decalDatasCount + m_LocalVolumetricFogCount;
+                m_TotalLightCount = m_GpuLightList.lightsCount + m_lightList.envLights.Count + decalDatasCount + m_LocalVolumetricFogCount;
```

Comment:
@sebastienlagarde as explained in slack, this is a very nice thing that we also get in CPU.
This entire copy is removed, since this implementation has the light bounds and lights per view bounds in a single array (not using c# List<>).
As a consequence I have to pass an extra offset  in the GPU (the offset must be different from the light count, because is the light count that comes from the internal culler, the visibleLightCount). THis is because the processed lights could do extra culling.
Anyway let me know your thoughts and please review the GPU updates of this offse.t